app/views/gsa18f/_request_details.html.haml

```diff
@@ -0,0 +1,24 @@
+= simple_form_for @client_data_instance, html: { multipart: true }, remote: true do |f|
+  - i = 0
+  - @proposal.new_fields_for_display.each do |field|
+    - if i == 1
+      :plain
+        <div class="medium-up-2">
+  
+    .column{ id: field[:key] + '-wrapper'}
+      .detail-wrapper.row
+        .detail-display
+          %label.detail-element
+            = t("decorators."+@current_user.client_slug+"/work_order."+field[:key]) 
```

Comment:
Spaces either side of the `+` signs would help the readability a lot (and on line 17)